returnn/oggzip.py

```diff
@@ -40,6 +41,7 @@ def __init__(
         :param bool no_conversion: do not call the actual conversion, assume the audio files are already correct
         :param Path|str returnn_python_exe: file path to the executable for running returnn (python binary or .sh)
         :param Path|str returnn_root: file path to the RETURNN repository root folder
+        :param dict[str] rqmt: job's resources requirements dict
```

Comment:
type is `dict[str, Any]|None`, also note here that setting this will disable running the job as mini task